manifests/s/SKCommunications/NateOn/7.0.6.0/SKCommunications.NateOn.locale.ko-KR.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SKCommunications.NateOn
+PackageVersion: 7.0.6.0
+PackageLocale: ko-KR
+Publisher: SK Communications
+PackageName: NateOn

```

Comment:
```suggestion
PackageName: ³×ÀÌÆ®¿Â
```

Reply:
![image](https://user-images.githubusercontent.com/44674417/124686890-dcb02680-df0e-11eb-948a-1ee6791512c0.png)
@ItzLevvie

Reply:
That's what it shows up as in Control Panel for me; and Windows Package Manager requires the name to match whatever's shown in Control Panel for the best upgrsde experience.

Reply:
fixed